gke-deploy/go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/google/gofuzz v1.1.0 // indirect
 	github.com/json-iterator/go v1.1.9 // indirect
 	github.com/kubernetes-sigs/application v0.8.1
-	github.com/pkg/errors v0.9.1 // indirect
+		github.com/pkg/errors v0.9.1
```

Comment:
Interesting I wonder what happened here

Reply:
in gcs.go, errors pkg is used explicitly.